packages/editor-ui/src/plugins/i18n/locales/en.json

```
@@ -1095,5 +1094,97 @@
 			"missingWorkflow": "Missing workflow",
 			"arrayEmpty": "Must have an array of nodes"
 		}
-	}
+	},
+	"404_NOT_FOUND_MESSAGE": "Oops, couldn’t find that",
+	"BASIC_INFORMATION": "Basic Information",
+	"CHANGE_PASSWORD": "Change Password",
+	"CHECK_INBOX_AND_SPAM": "Please check your inbox (and perhaps your spam folder)",
+	"CONFIRM_DATA_HANDLING_AFTER_DELETION": "What should we do with their data?",
+	"CONFIRM_USER_DELETION": "Are you sure you want to delete this invited user?",
+	"CURRENT_PASSWORD": "Current password",
+	"DELETE": "Delete",
+	"DELETE_CONFIRMATION_MESSAGE": "Type “delete all data” to confirm",
+	"DELETE_CONFIRMATION_TEXT": "delete all data",
+	"DELETE_USER": "Delete {user}",
+	"DELETE_WORKFLOWS_CREDENTIALS": "Delete their workflows and credentials",
+	"EMAIL": "Email",
+	"EMAIL_INVITES_SENT": "An invite email was sent to {emails}",
+	"EMAIL_INVITES_SENT_ERROR": "Could not invite {emails}",
+	"EMAIL_SENT_TO": "To {email}",
+	"ERROR": "Error",
+	"FINISH_ACCOUNT_SET_UP": "Finish account setup",
+	"FIRST_NAME": "First name",
+	"FORGOT_MY_PASSWORD": "Forgot my password",
+	"GET_RECOVERY_LINK": "Email me a recovery link",
+	"GO_BACK": "Go back",
+	"GO_TO_EDITOR": "Go to editor",
+	"INVALID_EMAIL_ERROR": "{email} is not a valid email",
+	"INVITE": "Invite",
+	"INVITE_NEW_USERS": "Invite new users",
+	"INVITE_RESENT": "Invite resent",
+	"INVITE_USER": "Invite user",
+	"INVITE_X_USER": "Invite {count} user",
+	"LAST_NAME": "Last name",
+	"LOADING": "Loading",
+	"LOGGING_IN_ERROR": "Problem logging in",
+	"MEMBER_ROLE": "Member",
+	"MISSING_INVITE_TOKEN_ERROR": "Missing invite token",
+	"MISSING_TOKEN_ERROR": "Missing token",
+	"MISSING_USERID_ERROR": "Missing user id",
+	"NEW_EMAILS_TO_INVITE": "New user email address(es)",
+	"NEW_PASSWORD": "New password",
+	"NEXT": "Next",
+	"NO_SMTP_TO_SEND_EMAIL_WARNING": "Please contact your admin. n8n isn’t set up to send email right now.",
+	"NO_USERS_TO_INVITE": "No users to invite",
+	"OWNER_ACCOUNT_BENEFITS_MESSAGE": "By setting up an owner account, you can invite other users to join your instance. It also ensures your instance can’t be accessed without an account.",
+	"PASSWORD": "Password",
+	"PASSWORDS_MUST_MATCH_ERROR": "Passwords must match",
+	"DEFAULT_PASSWORD_REQUIREMENTS": "At least 8 characters with 1 number and 1 uppercase",
+	"PASSWORD_UPDATE_ERROR": "Problem changing the password",
+	"PASSWORD_UPDATE_SUCCESS": "Password updated successfully",
+	"PASSWORD_UPDATE_SUCCESS_MESSAGE": "You can now sign in with your new password",
+	"PERSONAL": "Personal",
+	"PERSONAL_SETTINGS": "Personal Settings",
```

```suggestion
	"PERSONAL_SETTINGS_TITLE": "Personal Settings",
```
(Or other specifier that fits.)

---

it does not change the translation if it's a title or label or button